launch/gnss_driver.launch.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription, SetEnvironmentVariable
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+from launch_helpers import to_urdf
+
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    """Simple launch description to start gnss driver."""
+
+    rover_config_dir = get_package_share_directory('rover_config')
+    
+    gnss_driver_cmd = Node(
+        package='nmea_navsat_driver',
+        executable='nmea_serial_driver',
+        parameters=[os.path.join(rover_config_dir,'config','nmea_serial_driver.yaml')]
+        )
+
+    return LaunchDescription([
+		# Set env var to print messages to stdout immediately
+        SetEnvironmentVariable('RCUTILS_LOGGING_BUFFERED_STREAM', '1'),
```

Comment:
This does not do what the comment says. See https://index.ros.org/doc/ros2/Tutorials/Logging-and-logger-configuration/

Since foxy, all log output goes to unbuffered stderr. This env var changes that to buffered output, which can improve performance slighty, but can delay the output. I think we should stick to the default here.

```suggestion
		# Set env var to print messages to stdout immediately
        SetEnvironmentVariable('RCUTILS_LOGGING_BUFFERED_STREAM', '1'),
```

Reply:
Nicely spotted!
I assume your commit suggestion (currently it suggests leaving it how it is) is to remove the the command? 

I guess we should then change that in all launch file. The performance loss should not be too big, since even our main launch files, don't print data continuously.
What do you think?

Reply:
Ah damn, yeah that was supposed to be my suggestion :D

I think the more we stick to the defaults, the better. You got this from navigation2, right? Is there any mention why they put this?

Reply:
Yeah, I investigated it today and saw that it was just a PR from someone that changed it mainly to get rid of a warning message.

https://github.com/ros-planning/navigation2/issues/1865

Imo. it's fine to remove it since it's good to use defaults, it keeps it clean and our rovers have high performance PCs.